config/token.go

```diff
@@ -47,10 +47,10 @@ func GetSigningKey() string {
 }
 
 func ValidateSigningKey(signingKey string, authStrategy string) error {
-	if authStrategy != AuthStrategyAnonymous && (len(signingKey) == 0 || signingKey == "kiali") {
-		// "kiali" is a well-known signing key reported in a CVE. We ban it's usage.
-		// An empty key is also just not allowed.
-		return errors.New("signing key for login tokens is invalid")
+	if authStrategy != AuthStrategyAnonymous {
+		if len(signingKey) != 16 && len(signingKey) != 24 && len(signingKey) != 32 {
```

Comment:
What's the reason for these specific lengths i.e. 16, 24, and 32?

Reply:
The AES-GCM cipher does not like other lengths. It's tied to the key length of the algorithm.